source/crud/tests/unified/updateMany-hint-serverError.yml

```diff
@@ -0,0 +1,120 @@
+# This file was created automatically using mongodb-spec-converter.
+# Please review the generated file, then remove this notice.
+
+description: updateMany-hint-serverError
+schemaVersion: '1.1'
+createEntities:
+  -
+    client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      uriOptions:
+        retryReads: false
+      observeEvents:
+        - commandStartedEvent
+  -
+    database:
+      id: &database0 database0
+      client: client0
+      databaseName: &database_name crud-v2
+  -
+    collection:
+      id: &collection0 collection0
+      database: database0
+      collectionName: &collection_name test_updatemany_hint
+runOnRequirements:
+  -
+    minServerVersion: 3.4.0
+    maxServerVersion: 4.1.9
+initialData:
+  -
+    collectionName: *collection_name
+    databaseName: *database_name
+    documents:
+      -
+        _id: 1
+        x: 11
+      -
+        _id: 2
+        x: 22
+      -
+        _id: 3
+        x: 33
+tests:
+  -
+    description: 'UpdateMany with hint string unsupported (server-side error)'
+    operations:
+      -
+        object: *collection0
+        name: updateMany
+        arguments:
+          filter: &filter
+            _id:
+              $gt: 1
+          update: &update
+            $inc:
+              x: 1
+          hint: _id_
+        expectError:
+          isError: true
+    expectEvents:
+      -
+        client: *client0
+        events:
+          -
+            commandStartedEvent:
+              command:
+                update: *collection_name
+                updates:
+                  -
+                    q: *filter
+                    u: *update
+                    multi: true
+                    hint: _id_
+                    upsert:
+                      $$unsetOrMatches: false
+    outcome: &outcome
+      -
+        collectionName: *collection_name
+        databaseName: *database_name
+        documents:
+          -
+            _id: 1
+            x: 11
+          -
+            _id: 2
+            x: 22
+          -
+            _id: 3
+            x: 33
+  -
+    description: 'UpdateMany with hint document unsupported (server-side error)'
+    operations:
+      -
+        object: *collection0
+        name: updateMany
+        arguments:
+          filter: *filter
+          update: *update
+          hint:
+            _id: 1
+        expectError:
+          isError: true
+    expectEvents:
+      -
+        client: *client0
+        events:
+          -
+            commandStartedEvent:
+              command:
+                update: *collection_name
+                updates:
+                  -
+                    q: *filter
+                    u: *update
+                    multi: true
+                    hint:
+                      _id: 1
+                    upsert:
+                      $$unsetOrMatches: false
+    outcome: *outcome
```

Comment:
I noticed most files had a trailing newline. Should all of them, or is this something being inherited from the original input file?

Reply:
The trailing newlines were added by PhpStorm whenever I made manual changes to the files. I'll ensure every file has a trailing newline.